pkg/virtctl/vm/vm.go

```diff
@@ -511,6 +511,152 @@ func expandVirtualMachine(namespace string, virtClient kubecli.KubevirtClient, o
 	return nil
 }
 
+func startVirtualMachine(vmiName, namespace string, virtClient kubecli.KubevirtClient, dryRunOption *[]string) error {
+	err := virtClient.VirtualMachine(namespace).Start(context.Background(), vmiName, &v1.StartOptions{Paused: startPaused, DryRun: *dryRunOption})
+	if err != nil {
+		return fmt.Errorf("Error starting VirtualMachine %v", err)
+	}
+
+	return nil
+}
+
+func stopVirtualMachine(vmiName, namespace string, virtClient kubecli.KubevirtClient, dryRunOption *[]string) error {
+	if gracePeriodIsSet(gracePeriod) {
+		if !forceRestart {
+			return fmt.Errorf("Can not set gracePeriod without --force=true")
+		} else {
```

Comment:
nit: just avoid this else, you are already returning in the  if statement, we can avoid an additional nested block